vignettes/tidyverse_translation.Rmd

```diff
@@ -275,7 +275,45 @@ You can find a list of all the select helpers with `?data_select`.
 
 ## Extracting {#extracting}
 
-<!-- TODO: data_extract and dplyr::pull -->
+<!-- TODO: improve motivation for data_extract -->
```

Comment:
TBH, I don't think we need to motivate these functions here, since this vignette is just to set up translation with _tidyverse_. 

I had initially added sentences like these, but have already removed them.